### ISSUE TITLE
Provide navigation links to Get started from the Services Portal

### ISSUE THREAD

**Issue by timng-bcgov** [bcgov/api-services-portal#1169]
## Priority
Medium

## User Story

**As a** new or existing API provider,<br>
**I want**  to access the information provided on the Get started page<br>
**so that** I can create and configure Gateway services on the Services Portal.

### ENV

- [x] `DEV`
- [x] `TEST`
- [ ] `PROD`

### TESTCASE

Login to the API Services Portal.

### EXPECTED

There is/are link(s) to the Get started page from the Gateways manager.

### ACTUAL

There are no existing links to the Get started page if you are an existing user of the Services Portal (user with 1 or more Gateways).

## Notes

We heard from users during ns-to-gw user acceptance testing that they found the information on the Get started page useful as a reference. The majority of users would like to access this page once they navigate away from it in the Services Portal. We'll need to discuss more about the navigational patterns that we can design to address this issue. One possible solution is to revise the cards on the My Gateways manager, specifically the "Need to create a new Gateway?" item:

![image](https://github.com/user-attachments/assets/643f4183-8b37-433f-8f10-52d3a727c275)

**Comment by rustyjux** [comment]
If we are giving users a way back to Get Started, we might want to revise the 'You have Gateways' banner (since it was only made with the new/first time user in mind).

This could include one of:
- hide the banner
- remove the open in new tab (`target="_blank"`) for the link to My Gateways

I'm not sure what would be the best way to determine if we had a returning user vs first time user, if we wanted to make elements conditional based on that.

**Comment by timng-bcgov** [comment]
The latter option was a suggestion we heard in user testing that's captured in #1166 . I noted the link to this issue in the notes there as well. Which elements (other than the 'You have Gateways' banner) do you think we should make conditional?

**Comment by rustyjux** [comment]
Right now I'd say some language like 'your first Gateway' (as in 'Steps to create and configure your first Gateway') might want to change but...

**Before naming specific changes, how about we spell out the purpose for the Get Started page, both for the first time around and if users revisit?** In my mind, this page might be less confusing on a return visit the less that changes. Also, there are some style choice we use here (specifically 'our') which I find feel inappropriate and out of sync with the rest of our content (except for CMS Lite https://www2.gov.bc.ca/gov/content/data/finding-and-sharing/api-management).

Another revision for Get Started (which could be a separate issue) would be to change the page header/title from 'My Gateways' - this doesn't reflect the page content at all, except that you are redirected there from the Gateways tab (which is now 'My Gateways' once you open the page, but we need to revisit the whole 'My' thing).

**Comment by timng-bcgov** [comment]
Just so that I understand correctly, we are proposing to change the copy on the page with regards to whether the user is a first-time user (no Gateways) or an existing user (1 or more Gateways)?

I agree that we should rename the 'My Gateways' heading on the page. Can we add it to #1143?

**Comment by rustyjux** [comment]
> Just so that I understand correctly, we are proposing to change the copy on the page with regards to whether the user is a first-time user (no Gateways) or an existing user (1 or more Gateways)?

The rough original idea for this page was: give new API provider users a brief welcome to the Portal and a concise set of instructions to follow to gain experience with creating a Gateway (inc. Gateway Service, Product, and Dataset).

We can change components, copy, whatever! _if_ that seems necessary or helpful to suit the context in which users are viewing the page a second time.
 
Right now some page elements change based on the user state - we added the 'You have Gateways' banner (and removed the 'No gateways' block) so that if users happen to stay on this page after creating a Gateway (as instructions direct them to do), they have some indicators that things have changed from when they first loaded the page.

> I agree that we should rename the 'My Gateways' heading on the page. Can we add it to #1143?

Good call - I added Get Started to #1143

**Comment by timng-bcgov** [comment]
So with regards to what we discussed in sprint planning, how does this look for a proposed solution that can be accomplished this sprint?

- Change the 'My Gateways' heading on this page to 'Get started': #1143
- Agree on content changes ('our') and revise if necessary
- Change the behaviour of the link in 'Your existing Gateways' to open the Gateways manager in the same browser tab: #1166
- Revise the Gateways manager (/manager/gateways/list) to include a link to Get started in addition to the DevHub 'Create a Gateway' page in the 'Need to Create a new Gateway?' card:

![image](https://github.com/user-attachments/assets/91212352-bf16-42ea-b2fb-f98a9601a1f8)

All of these items are open to discussion, just writing this down so that the next steps can be fleshed out.

**Comment by timng-bcgov** [comment]
I see your point about having the choice of two links and how it might be suboptimal. I placed the link there for visibility and because it addresses the need to create a new Gateway. We could very well present a separate card on the Gateways list (I made some small changes to the copy while mocking up the design):

![image](https://github.com/user-attachments/assets/336b4aa2-9b8e-4923-ac66-96920c27fb69)

Another solution I'm considering in parallel is to include a link to **Get Started** from the **Help** dropdown.
